src/platforms/xamarin.ios/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build">
   <PropertyGroup>
     <AssemblyVersion>2.0.5.0</AssemblyVersion>
-    <TargetGroup>xamarin.ios</TargetGroup>
+    <TargetFramework>xamarin.ios</TargetFramework>
```

Comment:
I'm a bit surprised these didn't need the workarounds you used here: https://github.com/dotnet/standard/pull/1040/files#diff-4b0afd307f4db18276c29e6703aaa7d5R7

Reply:
I use them here. https://github.com/dotnet/standard/pull/1040/files#diff-54a81c90bf42ff99e0c8d2e04f436db6R11